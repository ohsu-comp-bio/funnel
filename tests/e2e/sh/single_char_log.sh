# Test that the streaming logs pick up a single character.
# This ensures that the streaming works even when a small
# amount of logs are written (which was once a bug).
id=`funnel run 'echo a'`
funnel task wait $id
funnel task get --view FULL $id
