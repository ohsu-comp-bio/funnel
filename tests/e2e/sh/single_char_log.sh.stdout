{
	"state": "COMPLETE",
	"name": "Funnel run: echo a",
	"resources": {

	},
	"executors": [
		{
			"imageName": "alpine",
			"cmd": [
				"echo",
				"a"
			],
			"workdir": "/opt/funnel",
			"stdout": "/opt/funnel/outputs/stdout-0",
			"stderr": "/opt/funnel/outputs/stderr-0"
		}
	],
	"logs": [
		{
			"logs": [
				{
					"stdout": "a\n",
				}
			],
		}
	]
}
